src/python/grpcio_observability/grpc_observability/__init__.py

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from grpc_observability._gcp_observability import GCPOpenCensusObservability
+from grpc_observability._open_telemetry_observability import (
+    OpenTelemetryObservability,
+)
+from grpc_observability._open_telemetry_plugin import OpenTelemetryPlugin
 
-__all__ = ("GCPOpenCensusObservability",)
+__all__ = ("OpenTelemetryObservability", "OpenTelemetryPlugin")
```

Comment:
Is `OpenTelemetryObservability` tied to GCP? Are other languages including "GCP" in the name of their equivalent API?